fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -549,6 +551,63 @@ class ShardManager {
 		return shard.get();
 	}
 
+	std::vector<std::string> removeRange(KeyRange range) {
```

Comment:
Can you add a comment what this method does and what it returns?

Reply:
Added some comments in IKeyValueStore.h